chokuretsu/guide/play-game.html

```diff
@@ -0,0 +1,222 @@
+                    <ol>
+                        <li>Download the <a href="https://www.dropbox.com/s/bei4xcrb5gueax0/Firmware%20Dumper.zip?dl=1"
+                                            target="_blank" class="exclude-device-href">DS Firmware dumper</a> and
+                            extract the <code>dsbf_dump.nds</code> file
+                        </li>
+                        <li>Load the DS firmware dumper .nds file onto your Flash cartridge's memory in the correct spot
+                            using its' SD card or connector cable.
+                        </li>
+                        <li>Insert the flash cartridge into your Nintendo DS console.</li>
+                        <li>Turn on your system and run the <code>dsbf_dump.nds</code> application</li>
+                        <li>Wait for the firmware dump to complete. Then, power off your Nintendo DS, remove the cart
+                            and insert the flash cart's SD card into your computer (or connect it with the correct
+                            cable).
+                        </li>
+                        <li>Your flash cartridge's SD card/storage will now contain <code>BIOSNDS7.ROM</code>, <code>BIOSNDS9.ROM</code>
+                            and a <code>.BIN</code> file starting with the letters "FW". These are your Nintendo DS'
+                            ARM7 and ARM9 BIOS files and its firmware respectively.
+                        </li>
+                    </ol>
+                </li>
+                <li>Copy your ARM7, ARM9 and firmware files into the same directory as where you downloaded MelonDS
+                    to.
+                </li>
+                <li>Open MelonDS, then on the top bar menu click "Config" <i class="fa-solid fa-arrow-right"></i> "Emu
+                    settings"
+                </li>
+                <li>On the "General" tab, ensure Console Mode is set to "DS". Click through to the tab named "DS-mode"
+                </li>
+                <li>On each of the required BIOS and firmware files, select the ARM7 <code>.ROM</code>, ARM9
+                    <code>.ROM</code> and firmware <code>.BIN</code> files you ripped from your system.
+                </li>
+            </ol>
+            <b>Playing the game:</b>
+            <p>With MelonDS setup, all you need to do in order to play the game is select "File" <i
+                    class="fa-solid fa-arrow-right"></i> "Open ROM" in the top menu bar and choose the ROM file you
+                patched earlier.</p>
+            <p>MelonDS has a number of advanced features, such as configuring controllers and tweaking display
+                settings. You can check out the <a href="https://melonds.kuribo64.net/" target="_blank"
+                                                   class="exclude-device-href">MelonDS Website</a> for more information.
+            </p>
+
+            <h4>Wii U Virtual Console setup</h4>
+            <p>TBD :-)</p>
```

Comment:
[Injectiine](https://gbatemp.net/threads/original-injectiine-wii-u-virtual-console-injector.491386/) is pretty good from what I've heard but I haven't done any VC injection myself so I'm unsure what's best.

Reply:
I'll see what I can do, the Wii U is in a storage container somewhere and I'm not able to get it out right now 😅. Might be a case of we write what we can from the internet and we'll see if we can get a tester to try out the guide.

Reply:
I have a Wii U out and already hacked so I can def try it out. I'll let you know.